magpie/db.py

```diff
@@ -129,14 +144,10 @@ def is_database_ready(db_session=None):
     table_names = inspector.get_table_names()
 
     for _, obj in inspect.getmembers(models):
-        if inspect.isclass(obj):
-            # noinspection PyBroadException
-            try:
-                curr_table_name = obj.__tablename__
-                if curr_table_name not in table_names:
-                    return False
-            except Exception:
-                continue
+        if inspect.isclass(obj) and hasattr(obj, "__tablename__"):
+            if obj.__tablename__ not in table_names:
+                LOGGER.error("Database table (or its associated parent) is missing for '{!s}' object".format(obj))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use % formatting in logging functions and pass the % parameters as arguments](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)